mainnet/docker-compose.yml

```diff
@@ -1,8 +1,8 @@
-version: '3.8'
+version: '3'
```

Comment:
no working for me with 3.8 ? should revert ?

Reply:
@andreibancioiu

Reply:
It's okay, do not revert.